### DISCUSSION TITLE
Список доступных -D параметров для вызова -jar

### DISCUSSION THREAD

**Discussion by d-mok-pvt** [qa-guru/allure-notifications#185] [Q&A] [answered]
В readme.md нашёл упоминание только -DconfigFile

Нашёл примеры реализации с работающими: 
-DreportLink
-DprojectName
-Dcomm
-Denv
-Dapp.bot.chat

Где можно найти список остальных поддерживемых параметров? Например для allureFolder, enableChart.

**Comment by KaterinaFixter** [comment]
Hey, did you get the reportLink to work at command line?

im trying the following but reportLink comes as blank/null, the config file is being picked up as thats where the slack api key is, but I left the reportLink blank in config and wanted to dynamically generate that from the github actions upload artifacts-url

${{ steps.upload-detox-allure-test-report.outputs.artifact-url }}

`java \
  "-DprojectName=Detox E2E Tests" \
  "-Denv=Development" \
  "-Dcomm=Scheduled execution of Detox E2E tests" \
  "-DreportLink=https://www.test.com/" \
  "-DconfigFile=notifications/config.json" \
  -jar notifications/allure-notifications-4.6.1.jar`

**Reply by d-mok-pvt** [reply]
No, I've just set up the config in the job before sending the notification. 
```shell
      - name: Set Allure config
        run: |
          echo '{
            "base": {
              "logo": "",
              "project": "'${{ env.TEST_PROJECT }}'",
              "environment": "'${{ env.ENVIRONMENT }}'",
              "comment": "'${{ env.COMMENT }}'",
              "reportLink": "'${{ env.ALLURE_JOB_RUN_URL }}'",
              "language": "en",
              "allureFolder": "allureReport",
              "enableChart": '${{ env.ENABLE_CHART }}'
            },
            "slack": {
              "token": "'${{ env.SLACK_TOKEN }}'",
              "chat": "'${{ env.SLACK_CHANNEL }}'",
              "replyTo": ""
            }
          }' > /home/runner/allure-config.json
```

**Reply by valfirst** [reply, marked as answer]
Only `configFile` system property is supported for now
